### DISCUSSION TITLE
Upper column name is recognized as an error

### DISCUSSION THREAD

**Discussion by mmcharles** [dbeaver/dbeaver#22761] [Q&A] [answered]
Hi, I can't understand why if I write a column name in upper letters it is recognized as a syntax error and is consequently colored red.

In the following example you can see how the same column is colored differently if written in lowercase or uppercase

![image](https://github.com/dbeaver/dbeaver/assets/80979938/4689cace-6279-450d-9516-c4488eb4b928)

**Comment by E1izabeth** [comment]
What DBeaver version do you use? What database is it?

**Reply by mmcharles** [reply]
DBeaver version: 23.3.2.202401100101
DataBase: PostgreSQL 12.10

**Reply by E1izabeth** [reply, marked as answer]
Please try DBeaver 23.3.4 version. Most probably, it's fixed there

**Reply by mmcharles** [reply]
Thanks, the problem is solved with the suggested version.

![image](https://github.com/dbeaver/dbeaver/assets/80979938/ad9724ec-8071-412a-b24e-d1ad4c9b5a6a)
